ruby/lib/helix_runtime/cli/bootstrap.rb

```diff
@@ -43,6 +43,18 @@ def add_gitignore
         template "gitignore", "#{base_path}/.gitignore"
       end
 
+      def add_publish_to_rubygems_file
+        template "publish_to_rubygems.rb", "#{base_path}/.scripts/publish_to_rubygems.rb"
+      end
+
+      def add_travis_yml_file
+        template ".travis.yml", "#{base_path}/.travis.yml"
```

Comment:
Doesn't technically need to be a template since there are no variables in the file, but this also probably doesn't hurt.